python/detail.cpp

```diff
@@ -51,7 +51,7 @@ void init_detail(py::module &m) {
       py::arg("data") = Variable{},
       py::arg("coords") = std::map<Dim, Variable>{},
```

Comment:
So before, would this always be setting aligned coords, but `.coords` would return both aligned and unaligned?
Seems it's more consistent now.

Reply:
Well, not always: We had this weird but also inconsistent behavior that `ds.coord['x'] = x` would insert and aligned coord, but `ds['a'].coords['x'] = x` added an unaligned coord. I will push an update that actually cleans up this meachnism.